### PR TITLE
Fix dockerfile for production

### PR DESCRIPTION
**Pull request by promehul**
###### Explanation About What Code Achieves:

  - Explanation

###### Screenshots/GIFs:

  - Screenshot

###### Steps To Test:

  - Steps

###### TODOs:

  - None

### CODE REVIEW

File: docker/production/app/Dockerfile [hunk 1]
```diff
@@ -37,45 +34,51 @@ RUN apt-get update --fix-missing \
...
+    build-essential \
+    graphviz-dev \
+    \
     # Remove cache
     && rm -rf /var/lib/apt/lists/*
 
 COPY . /app
 
-RUN pip3 install /app
+RUN pip3 install -e /app
```
Comment (achillesrasquinha) [inline docker/production/app/Dockerfile]: Why this change?

**Review by achillesrasquinha — APPROVED** [review]

File: docker/production/app/Dockerfile (hunk 1 above)
Comment (promehul) [reply docker/production/app/Dockerfile]: It was installed as editable in development as well. Does it make any difference?
